src/curated-corpus/components/RemoveSectionItemForm/RemoveSectionItemForm.tsx

```diff
@@ -57,12 +57,16 @@ export const RemoveSectionItemForm: React.FC<
       onSubmit(values, formikHelpers);
     },
   });
+  // remove "ML" from list of reasons, its only used by ML process
+  const filteredReasons = Object.values(SectionItemRemovalReason).filter(
+    (reason) => reason !== SectionItemRemovalReason.Ml,
+  );
```

Comment:
I wonder whether a cleaner solution would be to introduce two different sets of removal reasons, for example `EditorialSectionItemRemovalReason` that's extended with `ML` to form `SectionItemRemovalReason`. That way the special treatment of the `ML` can be re-used.